src/routes/modifyGame.ts

```diff
@@ -0,0 +1,68 @@
+import { getClient } from '../utils/databaseConnection'
+import { rateLimiter } from '../utils/rateLimiter'
+import { celebrate, Joi, Segments } from 'celebrate'
+import { verifyFCMToken } from '../utils/firebase'
+
+import express, { type Router } from 'express'
+const router: Router = express.Router()
+
+router.post(
+  '/modifyGame',
+  rateLimiter,
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      creatorToken: Joi.string()
+        .required()
+        .min(1)
+        .max(250)
+        .label('creatorToken'),
+      smallBlind: Joi.number().required().min(1).label('smallBlind'),
+      startingFunds: Joi.number().required().min(1).label('startingFunds'),
+    }),
+  }),
+  async (req, res) => {
+    if (
```

Comment:
I think you copied this if statement from kick player and it doesn't really make sense